app/components/footer_home/_footer_home.html.erb

```diff
@@ -41,7 +41,6 @@
             <li><%= mail_to "kim@placecal.org", "Email us" %></li>
             <li><%= link_to "Get PlaceCal", join_path %></li>
             <li><%= link_to "Twitter", 'https://twitter.com/PlaceCal' %></li>
-            <li><%= link_to "Facebook", 'https://facebook.com/PlaceCal' %></li>
```

Comment:
Oh oops I missed this -- I don't think this ticket covers removing a link to our facebook page